Pycloud Deployer/modules.py

```diff
@@ -0,0 +1,277 @@
+import re
+import getpass
+from config import location, subscription_id
+
+
+def validate_group_name(group_name):
```

Comment:
In validate_group_name and get_vnet_or_subnet_name, the length limit is set to 90 characters, while in get_nsg_name and get_server_name, different limits are used. Ensure that these limits are consistent with Azure's actual requirements for these resources.